### ISSUE TITLE
1.6.1a server crash after load - eio conduit

### ISSUE THREAD

**Issue by forry** [Nomi-CEu/Nomi-CEu#422]
### Nomi CEu Version

1.6.1a

### Launcher

curse forge

### Configurations or Mods Changed

astral  astralsorcery-1.12.2-1.10.27
stepup  StepUp-1.0.2-mc1.10-1.12
ftb ultimine  ftb-ultimine-1202.3.5
tramplestopper  tramplestopper-1.12.2-1.2.0.5-universal
swing through grass  stg-1.12.2-1.2.3

### Environment

server

### Mode

Normal Mode

### What Happened

Crash instantly after load and trying to tick entities

### Expected Behavior

load

### Reproduction Steps

start launch.bat and wait for server to load and crash

### Logs

[crash-2023-09-06_18.17.22-server.txt](https://github.com/Nomi-CEu/Nomi-CEu/files/12540216/crash-2023-09-06_18.17.22-server.txt)

**Comment by IntegerLimit** [comment]
How did you setup the server? This was a bug in 1.6. Try making new instance instead of pressing update on server.

**Comment by IntegerLimit** [comment]
Re-open or comment on this if you cannot fix the issue.

**Comment by forry** [comment]
> How did you setup the server? This was a bug in 1.6. Try making new instance instead of pressing update on server.

I have downloaded the zip with the server files for the last 1.6 version from curse forge web page - nomi-ceu-server-1.6.zip. Unpacked it to a new dir. Then to an unpacked dir, I have copied other mods we use, the folder with the world, and all these server.properties and json files with whitelist, ops, etc. So I would say it is close to a new instance.
